List: update Selection guide (#4911)
DevExpress/devextreme-documentation: concepts/05 UI Components/List/25 Selection/01 User Interaction.md

> To enable selection in the List component, use the [selectionMode](/api-reference/10%20UI%20Components/dxList/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode') property to specify one of selection modes.

Reviewer: ```suggestion
To enable selection in the List component, use the [selectionMode](/api-reference/10%20UI%20Components/dxList/1%20Configuration/selectionMode.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode') property to specify the selection mode.
```

---

> To select a List item, an end user clicks or taps it. Selected items become shaded. If you want to indicate selected items, set the [showSelectionControls](/api-reference/10%20UI%20Components/dxList/1%20Configuration/showSelectionControls.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls') property to `true`. This setting adds a check box to each item on the List. Enable this option if you use the *"all"* selection mode. Otherwise, the *"Select All"* check box will be missing.

Reviewer: ```suggestion
To select a List item, a user should click or tap it. Selected items become shaded. If you want to indicate selected items, set the [showSelectionControls](/api-reference/10%20UI%20Components/dxList/1%20Configuration/showSelectionControls.md '/Documentation/ApiReference/UI_Components/dxList/Configuration/#showSelectionControls') property to `true`. This setting adds a checkbox to each item on the List. Enable this option if you use the *"all"* selection mode. Otherwise, the *"Select All"* checkbox is not shown.
```

No consistency: users / end users, checkboxes / check boxes
